holoviews/tests/operation/test_downsample.py

```diff
@@ -20,16 +21,26 @@ def test_downsample1d_multi(plottype):
     assert N > downsample1d.width
 
     if plottype == "overlay":
-        figure = hv.Overlay([hv.Curve(range(N)), hv.Curve(range(N))])
+        figure = Overlay([Curve(range(N)), Curve(range(N))])
     elif plottype == "ndoverlay":
-        figure = hv.NdOverlay({"A": hv.Curve(range(N)), "B": hv.Curve(range(N))})
+        figure = NdOverlay({"A": Curve(range(N)), "B": Curve(range(N))})
```

Comment:
I have never been the fan of the way we write our test and the way we write our code differs. Makes it harder to copy/paste either from or to a notebook. 

Though, I'm curious is there any other reason for this change other than to align them with the rest?

Reply:
Nope, just aligning with the style elsewhere.